src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+                ]
+              },
+              {
+                "issueId": "2b3fcffe-5330-4766-bc51-392b6db5eedc",
+                "priority": "MEDIUM",
+                "state": "CLOSED",
+                "title": [
+                  "Anomaly: It was different from normal"
+                ]
+              },
+
+```
+  </Collapser>
+  <Collapser
+     className="freq-link"
+     id="cursor"
+     title="Cursor pagination"
+   >
+
+In order to paginate through your data, you must include the `nextCursor` field on your initial query.
+
+With cursor pagination, you continue to make requests through the result data until the `nextCursor` returned from the response comes back empty. An empty response means you've arrived at the end of your results.
+
+Here's an example:
+
+**Initial Request**
+
+```graphql
+{
+  currentUser {
+    account(id: YOUR_ACCOUNT_ID) {
+      aiIssues {
+        issues(cursor: "") {
+          issues {
+            issueId
+            priority
+            state
+            title
+          }
+          nextCursor
+        }
+      }
+    }
+  }
+}
+```
+
+**Initial Results**
+
+```json
+{
+  "data": {
+    "currentUser": {
+      "account": {
+        "aiIssues": {
+          "issues": {
+            "issues": [
+              {
+                "issueId": "bda33b52-c70b-47b9-abe3-9e859a394037",
+                "priority": "CRITICAL",
+                "state": "ACTIVATED",
+                "title": [
+                  "Signal lost for 5 minutes on 'Percent All Facet'"
+                ]
+              },
+              {
+                "issueId": "242c713c-7654-491c-808e-3aa7fc44812b",
+                "priority": "MEDIUM",
+                "state": "CREATED",
+                "title": [
+                  "Anomaly: Non-web throughput"
+                ]
+              },
+
+
+              {
+                "issueId": "f19c9803-fb24-4d5d-9c35-16f659bceb3a",
+                "priority": "MEDIUM",
+                "state": "CLOSED",
+                "title": [
+                  "Anomaly: Error rate"
+                ]
+              }
+            ],
+            "nextCursor": "=ATN"
+          }
+        }
+      }
+    }
+  },
+```
+
+Then, in your subsequent request, provide the cursor value from `nextCursor` previous reply, until the `nextCursor` is empty.
+
+**The subsequent request should like:**
+
+```graphql
+{
+  currentUser {
+    account(id: YOUR_ACCOUNT_ID) {
+      aiIssues {
+        issues(cursor: "=ATN") {
+          issues {
+            issueId
+            priority
+            state
+            title
+          }
+          nextCursor
+        }
+      }
+    }
+  }
+}
+```
+
+ </Collapser>
+  <Collapser
+     className="freq-link"
+     id="filter"
+     title="Filter issues"
+   >
+
+Filter issues by entity guides, entity types, issues priority or issues states or any combination of those attributes.
+
+```graphql
+"Filter issues"
+input FilterIssues {
```

Comment:
they seem to be using a new format here for customer-specific variables. usually we would use var tags, lik 

<var>STRING</var>

If we're going to change that, we'd probably need to change the tables at the top, as I think they were trying to define the values (the ones with ! at end) up above and use them below. But not entirely sure

Reply:
@neta-signifai are we changing the format here?